docs/general/folderstructure.rst

```diff
@@ -3,7 +3,7 @@ Folder Structure
 
 The following demonstrates a possible structure created to store intermediate and final files during a processor run.
 The listed files describe the output of the user configuration parameter ``measurement`` set to ``gamma``
-and all output annotation layers enabled (``annotation = dm, ei, em, id, lc, li, np, gs``).
+and all output annotation layers enabled (``annotation = dm, ei, em, id, lc, li, np, gs``), thus creating S1-NRB products.
```

Comment:
`
and all S1-NRB output annotation layers enabled (``annotation = dm, ei, em, id, lc, li, np, ratio``)
`
- `gs` has been renamed to `ratio`
- a new ORB-only layer `ld` has been introduced

Reply:
Done https://github.com/SAR-ARD/S1_NRB/pull/121/commits/a154de7b7274cf8fb809b7c087bd125ecff2cea3

Reply:
The look direction angle is not intended for NRB so your sentence is a little misleading. Enabling all layers is not creating an NRB product.

Reply:
You're right. How about:

>and **the following** output annotation layers enabled (``annotation = dm, ei, em, id, lc, li, np, ratio``), thus creating S1-NRB products.

And only mentioning `ld` as an option in the config file, but removing it as a default.

We can integrate ORB further into the docs once it has also been further integrated in the processor.

Reply:
sounds good :+1: